spec/factories/application_templates.rb

```diff
@@ -2,6 +2,7 @@
   factory :application_template do
     position
     active true
+    email 'steve@sharklazers.com'
```

Comment:
Yes. Factories should only contain the attributes which are necessary in order to make a valid instance.